openml-utils/src/test/java/com/feedzai/util/provider/AbstractProviderModelBaseTest.java

```diff
@@ -255,6 +268,33 @@ private void assertAllInstances(final List<List<Future<Integer>>> evaluationsLis
         return results;
     }
 
+    /**
+     * Classifies on of two dummy instances using a given model in parallel using a given {@link ExecutorService}.
+     * This method creates the same number of events for each dummy instance.
+     *
+     * @param model    Model to use to classify dummy instances.
+     * @param executor Executor to use to launch the threads that will classify each instance.
+     * @return A list of futures that will return the class of each instance.
+     * @since 0.2.0
+     */
+    private List<Future<double[]>> classDistTwoInstancesInParallel(final M model,
+                                                                   final ExecutorService executor) {
+        final CountDownLatch latch = new CountDownLatch(1);
+
+        final List<Future<double[]>> results = IntStream.range(0, this.maxNumberOfThreads)
+                .mapToObj(idx -> executor.submit(() -> {
+                    if (!latch.await(10, TimeUnit.SECONDS)) {
```

Comment:
does this have to be "10" seconds? could it be 3 or something smaller? can it be a constant (since it's now being used in 2 places)?